internal/time/duration.go

```diff
@@ -63,5 +64,11 @@ func (d *ReadableDuration) UnmarshalJSON(data []byte) error {
 
 // Seconds returns ReadableDuration in seconds.
```

Comment:
shall we improve the godoc to include the fact that `Seconds` method on `ReadableDuration` would floor the value to seconds, discarding milliseconds or lower values?